src/main/default/classes/TriggerConfigService.cls

```diff
@@ -0,0 +1,115 @@
+public class TriggerConfigService {
+    public static ObjectConfig getConfig(String objName) {
+        String lowercaseObjName = objName.toLowerCase();
+        if (!configMetadata.keySet().contains(lowercaseObjName)) {
+            return null;
```

Comment:
this is the same as `configMetadata.get(lowercaseObjName);` where the key doesn't exists